telegram/src/test/kotlin/com/github/kotlintelegrambot/testutils/IOUtils.kt

```diff
@@ -1,9 +1,10 @@
 package com.github.kotlintelegrambot.testutils
 
 import java.io.File
+import java.net.URLDecoder
 
 inline fun <reified T> getFileAsStringFromResources(resName: String): String =
-    T::class.java.getResource("/$resName").readText()
+    getFileFromResources<T>(resName).readText()
 
 inline fun <reified T> getFileFromResources(resName: String): File =
-    File(T::class.java.getResource("/$resName").file)
+    File(URLDecoder.decode(T::class.java.getResource("/$resName")!!.file, "UTF-8"))
```

Comment:
Do we need the `URLDecoder.decode` part?

Reply:
Yes, otherwise if the path to the project folder on developer machine contains spaces, tests calling this method will fail `java.io.FileNotFoundException`:

>E:\Git%20repositories\kotlin-telegram-bot\telegram\build\resources\test\short.ogg (Системе не удается найти указанный путь)

Reply:
I just realized that Kotlin stdlib has a nicer way to do this:

```kt
T::class.java.getResource("/$resName")!!.file.decode()
```